content/blog/2018-09-14-smapr.md

```diff
@@ -0,0 +1,637 @@
+---
+slug: "smapr"
+title: Mapping the 2018 East Africa floods from space with smapr
+package_version: 0.1.2
+authors:
+  - name: Max Joseph
+    url: https://github.com/mbjoseph
+date: 2018-09-14
+categories: blog
+topicid: 925
+tags:
+- r
+- community
+- software
+- review
+- onboarding
+- package
+- data-access
+- geospatial
+---
+
+Hundreds of thousands of people in east Africa have been displaced and hundreds have died as a result of torrential rains which ended a drought but saturated soils and engorged rivers, resulting in extreme flooding in 2018.
```

Comment:
This is an excellent intro - makes me want to read on and frames the issue for which the packages are developed & used

Reply:
Thanks @stefaniebutland! Glad you like it.